Broadcom/symantec-endpoint-protection/ingest/parser.yml

```
@@ -47,13 +65,28 @@ stages:
           observer.product: "Symantec Endpoint Protection"
           event.kind: "event"
           event.category: ["malware"]
-          event.type: ["info"]
           event.reason: "{{parsed_event.message.reason}}"
           event.action: "{{parsed_event.message.action}}"
           host.ip: "{{parsed_event.message.host_ip}}"
-          host.hostname: "{{parsed_event.message.host_name}}"
-          host.name: "{{parsed_event.message.host_name}}"
+          host.hostname: "{{parsed_event.message.host_hostname}}"
+          host.name: "{{parsed_event.message.host_name or parsed_event.message.host_hostname}}"
```

```suggestion
          host.name: "{{parsed_event.message.host_name}}"
```

You could skip the last part for readability purpose (the finalizer will do it automatically)